QuestionAnswering/QA_ICL/backend/fastapi_app/static/css/form-styles.css

```diff
@@ -0,0 +1,80 @@
BQUUsTUFBTSxHQUNyQjs7QUFqQ0wsQUFtQ0ksV0FuQ08sQUFtQ04sV0FBVyxDQUFDLEtBQUssQ0FBQSxFQUNkLFlBQVksRUFBRSxDQUFDLEdBQ2xCOztBQUlMLEFBQ0ksc0JBRGtCLENBQ2xCLFdBQVcsQUFBQSxXQUFXLENBQUMsRUFDbkIsWUFBWSxFQUFFLEdBQUcsRUFDakIsY0FBYyxFQUFFLE1BQU0sR0FDekI7O0FBSkwsQUFLSSxzQkFMa0IsQ0FLbEIsV0FBVyxHQUFHLEtBQUssQ0FBQyxFQUNoQixPQUFPLEVBQUUsWUFBWSxHQUN4Qjs7QUFJTCxBQUFBLGNBQWMsQ0FBQyxFQUNYLE9BQU8sRUFBRSxXQUFXLEVBQ3BCLFFBQVEsRUFBRSxNQUFNLEVBQ2hCLGFBQWEsRUFBRSxHQUFHLEVBQ2xCLFdBQVcsRUFBRSxJQUFJLEVBQ2pCLE1BQU0sRUFBRSxHQUFHLENBQUMsS0FBSyxDQTlMRCxJQUFJLEdBMk52Qjs7QUFsQ0QsQUFPSSxjQVBVLENBT1YsS0FBSyxDQUFBLEFBQUEsSUFBQyxDQUFELEtBQUMsQUFBQSxFQUFZLEVBQ2QsUUFBUSxFQUFFLFFBQVEsRUFDbEIsVUFBVSxFQUFFLE1BQU0sRUFDbEIsT0FBTyxFQUFFLElBQUksR0FDaEI7O0FBWEwsQUFhSSxjQWJVLENBYVYsS0FBSyxDQUFDLEVBQ0YsT0FBTyxFQUFFLFlBQVksRUFDckIsTUFBTSxFQUFFLE9BQU8sRUFDZixPQUFPLEVBQUUsTUFBTSxFQUNmLE1BQU0sRUFBRSxDQUFDLEVBQ1QsV0FBVyxFQUFFLE1BQU0sRUFDbkIsS0FBSyxFQUFFLE9BQU8sRUFDZCxVQUFVLEVBQUUsMEJBQTBCLEdBQ3pDOztBQXJCTCxBQXVCSSxjQXZCVSxDQXVCVixLQUFLLEFBQUEsVUFBVSxBQUFBLFFBQVEsR0FBRyxLQUFLLENBQUMsRUFDNUIsVUFBVSxFQUFFLElBQUksRUFDaEIsS0FBSyxFQUFFLElBQUksR0FDZDs7QUExQkwsQUE0QkksY0E1QlUsQ0E0QlYsS0FBSyxBQUFBLFFBQVEsR0FBRyxLQUFLLENBQUMsRUFDbEIsS0FBSyxFQUFFLElBQUksRUFDWCxVQUFVLEVBQUUsSUFBSSxHQUNuQjs7QUFLTCxtQkFBbUI7QUFDbkIsQUFBQSxjQUFjLENBQUMsRUFDWCxRQUFRLEVBQUUsUUFBUSxFQUNsQixPQUFPLEVBQUUsV0FBVyxFQUNwQixPQUFPLEVBQUUsV0FBVyxFQUNwQixPQUFPLEVBQUUsSUFBSSxFQUNiLGtCQUFrQixFQUFFLFFBQVEsRUFDNUIscUJBQXFCLEVBQUUsTUFBTSxFQUM3QixrQkFBa0IsRUFBRSxNQUFNLEVBQzFCLGNBQWMsRUFBRSxNQUFNLEVBQ3RCLFNBQVMsRUFBRSxJQUFJLEVBQ2YsS0FBSyxFQUFFLElBQUksRUFDWCxNQUFNLEVBQUUsSUFBSSxFQUNaLFNBQVMsRUFBRSxLQUFLLEVBQ2hCLFVBQVUsRUFBRSxLQUFLLEVBQ2pCLE1BQU0sRUFBRSxpQkFBaUIsRUFDekIsZ0JBQWdCLEVBQUUsSUFBSSxFQUN0QixPQUFPLEVBQUUsSUFBSSxHQUNoQjs7QUFFRCxBQUFBLG9CQUFvQixDQUFDLEVBQ2pCLFFBQVEsRUFBRSxRQUFRLEVBQ2xCLGdCQUFnQixFQUFFLENBQUMsRUFDbkIsUUFBUSxFQUFFLENBQUMsRUFDWCxJQUFJLEVBQUUsQ0FBQyxFQUNQLE1BQU0sRUFBRSxpQkFBaUIsRUFDekIsVUFBVSxFQUFFLEtBQUssR0FDcEI7O0FBRUQsQUFBQSxvQkFBb0IsQ0FBQyxNQUFNLENBQUMsRUFDeEIsUUFBUSxFQUFFLFFBQVEsRUFDbEIsSUFBSSxFQUFFLENBQUMsRUFDUCxHQUFHLEVBQUUsQ0FBQyxFQUNOLEtBQUssRUFBRSxJQUFJLEVBQ1gsTUFBTSxFQUFFLElBQUksRUFDWixhQUFhLEVBQUUsR0FBRyxFQUNsQixVQUFVLEVBQUUsQ0FBQyxDQUFDLENBQUMsQ0FBQyxHQUFHLENBQUMsbUJBQW1CLENBQUMsS0FBSyxHQUNoRDs7QUFFRCxBQUFBLHNCQUFzQixDQUFDLEVBQ25CLEtBQUssRUFBRSxPQUFPLEVBQ2QsVUFBVSxFQUFFLE1BQU0sRUFDbEIsU0FBUyxFQUFFLEtBQUssR0FDbkI7O0FBRUQsQUFBQSx1QkFBdUIsQ0FBQyxFQUNwQixPQUFPLEVBQUUsV0FBVyxFQUNwQixPQUFPLEVBQUUsV0FBVyxFQUNwQixPQUFPLEVBQUUsSUFBSSxFQUNiLGdCQUFnQixFQUFFLE9BQU8sRUFDekIsYUFBYSxFQUFFLE9BQU8sRUFDdEIsZUFBZSxFQUFFLGFBQWEsRUFDOUIsVUFBVSxFQUFFLEdBQUcsR0FDbEI7O0NBRUQsQUFBQSxBQUFBLGVBQUMsQ0FBZ0IsT0FBTyxBQUF2QixFQUF5QixTQUFTLENBQUMsRUFDaEMsT0FBTyxFQUFFLElBQUksRUFDYixXQUFXLEVBQUUsTUFBTSxFQUNuQixhQUFhLEVBQUUsTUFBTSxHQUN4Qjs7Q0FFRCxBQUFBLEFBQUEsZUFBQyxDQUFnQixPQUFPLEFBQXZCLEVBQXlCLFNBQVMsR0FBRyxLQUFLLEdBQzNDLEFBQUEsZUFBQyxDQUFnQixPQUFPLEFBQXZCLEVBQXlCLFNBQVMsR0FBRyxRQUFRLENBQzlDLEVBQ0ksTUFBTSxFQUFFLFFBQVEsRUFDaEIsT0FBTyxFQUFFLFlBQVksR0FDeEIifQ== */
```

Comment:
This file is downloaded verbatim from http://kg.cmclinnovations.com/user/plugins/form/assets/form-styles.css and is committed to the repo as per your previous instruction about committing static resources to the source code to remove dependency on externally hosted resources. 

The React frontend for Marie doesn't require this file, and it is only here because the incomplete implementation of Zaha, which reuses the HTML template from previous iterations, references this file. 

Please advise what I should do to this file.
1) Remove the entire file
2) Remove only the commented text i.e. `/*# sourceMappingURL=data:application/json;...`